From 7d72ff00358174be24278fd53d5661df36e9d535 Mon Sep 17 00:00:00 2001
From: Euphyllia Bierque <bierque.euphyllia@gmail.com>
Date: Wed, 30 Jul 2025 12:18:11 +0200
Subject: [PATCH 8/8] Bump bstats

---
 gradle/libs.versions.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gradle/libs.versions.toml b/gradle/libs.versions.toml
index a9f9a6a6..cd3cf8e1 100644
--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -31,7 +31,7 @@ paperApi = "io.papermc.paper:paper-api:1.21.5-R0.1-SNAPSHOT"
 paperLib = "io.papermc:paperlib:1.0.8"
 
 dummypermscompat = "com.sk89q:dummypermscompat:1.10"
-bstats-bukkit = "org.bstats:bstats-bukkit:2.2.1"
+bstats-bukkit = "org.bstats:bstats-bukkit:3.1.0"
 
 prtree = "org.khelekore:prtree:1.5.0"
 
-- 
2.50.1.windows.1

